src/components/VelikanovKY/WindDirection.js

```diff
@@ -0,0 +1,38 @@
+ const WindDirection = (deg) =>{
+    let direction = 'No way';
+    deg = deg.deg  
```

Comment:
Выглядит странно. 
Может лучше сделать так:
 const WindDirection = (deg) =>{
```suggestion
    const WindDirection = ({ deg }) => {
    let direction = 'Undefined';
    deg = deg.deg  // Убрать
```

Reply:
Да, так лучше, но всё ещё не понятно, почему я должен писать deg.deg

Reply:
Потому что на вход идёт объект с пропсами: `const WindDirection = (props) => {`
Уже из пропсов достаём `deg`:
`props.deg` → используем деструктуризацию `({ deg }) => {`